docs/sdk/SDK_SMPTE_ST2110_RX_FLOW.md

```diff
@@ -0,0 +1,104 @@
+# Mesh Data Plane SDK – Example / SMPTE ST2110-22 Rx Flow
+
+The diagram in this document shows interaction between a user app and Mesh Data Plane in the video receive mode over SMPTE ST2110-22.
+
+### Rx flow stages
+1. Create client.
+1. Create connection.
+1. Configure connection.
+1. Establish Tx connection.
```

Comment:
Should that not be Rx, instead of Tx?

Reply:
Yep, my fault. Will update it.

Reply:
Fixed in eae8b083e0210567994eb379b5e228742451c44b.